src/main/java/com/oltpbenchmark/util/SQLUtil.java

```diff
@@ -109,20 +113,46 @@ public static Double getDouble(Object obj) {
         return (null);
     }
 
+    public static String clobToString(Object obj) {
+        try {
+            Clob clob = (Clob) obj;
+            return clob.getSubString(1, (int) clob.length());
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
     public static String getString(Object obj) {
         if (obj == null) {
             return (null);
         }
 
         if (obj instanceof String) {
             return (String) obj;
+        } else if (obj instanceof BigDecimal bigDecimal) {
+            return bigDecimal.toString();
         }
 
         LOG.warn("BAD BAD BAD: returning null because getString does not support {}", obj.getClass());
 
         return (null);
     }
 
+    private static final Class<?> ORACLE_TIMESTAMP;
+    private static final Method TIMESTAMP_VALUE_METHOD;
+    static {
+        Method timestampValueMethod;
+        Class<?> oracleTimestamp;
+        try {
+            oracleTimestamp = Class.forName("oracle.sql.TIMESTAMP");
+            timestampValueMethod = oracleTimestamp.getDeclaredMethod("timestampValue");
+        } catch (ClassNotFoundException | NoSuchMethodException e) {
+            oracleTimestamp = null;
+            timestampValueMethod = null;
+        }
+        TIMESTAMP_VALUE_METHOD = timestampValueMethod;
+        ORACLE_TIMESTAMP = oracleTimestamp;
+    }
```

Comment:
What is this code doing? Why is it necessary?

Reply:
+1, a few more comments would be nice

Reply:
Oracle support needed TIMESTAMP fields which corresponds to OJDBC-specific type `oracle.sql.TIMESTAMP` (not available in JDBC). 

I need to convert OJDBC timestamp to JDBC `java.sql.Timestamp` (with [timestampValue](https://docs.oracle.com/en/database/oracle/oracle-database/21/jajdb/oracle/sql/TIMESTAMP.html#timestampValue__) method) without having to import the type directly, hence the use of reflection with `Class.forName(String)` and `getDeclaredMethod(String)`. For Oracle profile the types will be initialized and used in `getTimestamp(Object)` and for other profiles the variables will be null and the conversion branch wouldn't be reachable.